_includes/search.html

```diff
@@ -0,0 +1,25 @@
+<script language="Javascript" type="text/javascript">
+  function my_search_google()
+  {
+    var query = document.getElementById("my-google-search").value;
+    window.open("http://google.com/search?q=" + query
+	+ "%20site:" + "http://theforeman.org");
+  }
+</script>
+
+<div class="container container-search">
+  <a class="pull-left" href="/feed.xml">
+    <img src="/static/images/rss.png" alt="RSS Feed" />
+    Get the RSS News Feed
```

Comment:
Actually, thinking about it again, I'm OK with it staying if we can get a proper heading above it.  What bothered me was that it seemed to be the page heading, so it distracted you from reading the page content.

What do you think of https://github.com/domcleal/theforeman.org/commit/f0358390aebc34cc97035998cc818b43d43dde70?  It reduces the font sizes again, but adds in a page heading which helps refocus the page IMHO.